CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## X.X.X (YYYY-MM-DD)
+
+-   Inline named object instances are now available when using completion lists, hover and other features like object instances are.
```

Comment:
j'ai pas compris le "are' a la fin...

Reply:
“de la même manière que les blabla le sont [déjà]”

Reply:
aaaah puree je voyais ca "Inline named object instances are now available when using completion lists," puis une autre partie de phrase "hover and other features like object instances are." du coup ca n'avait aucn sens :D

Reply:
je me demande si on ne pouvait pas rajouter une virgule avant "like" pour que ca soit plus evident. mais pas sure... enfin bref c'est comprehensible mtn de toute facon ^^